tools/coqmod/README.md

```diff
@@ -0,0 +1,72 @@
+# coqmod
+
+`coqmod` outputs Coq modules and OCaml libraries that a `.v` file depends on.
+
+It is intended to be a minimalistic replacement to `coqdep` for use by `dune`.
+
+Unlike `coqdep`, it does not scan directories making it lightweight in
+comparison.
+
+It has 3 `--format` modes:
+- `csexp` - canonical serial expresion (default)
+- `sexp` - pretty printed serial expression
+- `read` - human readable output
+
+When a single `.v` file is passed to `coqmod`, it will output the tokens
+corresponding to dependencies in the specified format.
+
+Here is the output for an `example.v` file:
+
+```coq
+  From A.B.C Require Import R.X L.Y.G Z.W.
+
+  Load A.B.X.
```

Comment:
That's not real syntax

Reply:
Then this is a bug in the coqdep lexer. I can fix it here and in coqdep, but is it worth fixing? It just seems that we accept more than possible. I will remove it from the test and the README.md however.

Reply:
FTR, I've stopped Load from being lexed as a `X.Y` and instead enforced a single `X`.